packages/geo-diary/src/utils/map-utils.ts

```diff
@@ -37,25 +37,23 @@ export const fetchDestinationInformation = ({
   })
 }
 
-export const getAppStateWithGeoCoords = (appState: AppState): Promise<AppState> => {
+export const getGeoCoords = (): Promise<Partial<AppState>> => {
   return new Promise((resolve) => {
     const hasGeoLocation = Boolean(navigator.geolocation)
 
-    if (!hasGeoLocation) return resolve(appState)
+    if (!hasGeoLocation) return resolve({})
 
     navigator.geolocation.getCurrentPosition(
       (position) => {
         return resolve({
-          ...appState,
-          hasGeoLocation: true,
           currentLat: position.coords.latitude,
           currentLng: position.coords.longitude,
         })
       },
       (error) => {
         const err = new Error(error.message)
         logger(err)
-        return resolve(appState)
+        return resolve({})
```

Comment:
should this reject?

Reply:
No because it would be a failure of the browser's native geo-location which is likely a network issue. In this case, the app falls back gracefully to a default location so I just want the app to carry on as normal. Hence, resolve the promise.